src/main/java/com/amazon/ion/IonSequence.java

```diff
@@ -426,15 +426,15 @@ public void add(int index, IonValue child)
      * </p>
      *
      * The implementation of {@link List<IonValue>} returned by this method
-     * implements {@link List#equals(Object)} and
-     * {@link List#equals(Object)} ()} per the specification of these methods.
+     * implements {@link List#equals} and {@link List#hashCode()}
```

Comment:
You double checked that `hashCode()` is consistent with `List`, I presume? Do we need to add a caveat to `hashCode()` that it is not consistent with `equals()`?